cmd/kubeadm/app/phases/upgrade/postupgrade_v18_19.go

```diff
@@ -31,6 +31,7 @@ import (
 )
 
 var v190 = version.MustParseSemantic("v1.9.0")
+var v190alpha3 = version.MustParseSemantic("v1.9.0-alpha.3")
```

Comment:
can we add a todo to consolidate this with vars in features dir?

Reply:
yes